### DISCUSSION TITLE
Does it work with LG climate without wall controls?

### DISCUSSION THREAD

**Discussion by phankhanhpvk** [JanM321/esphome-lg-controller#44] [Q&A] [answered]
Does it work with LG air conditioners without wall controls? Is there any way I can check. On the cold side motherboard have CN-REMO

**Comment by landrysik** [comment, marked as answer]
What do you mean by "without wall controls"? This project REPLACES the wired wall control unit. But I don't think there is any official list which units support wired wall controller. You have to try it. There is good chance that the unit is supported if there is a connector for the wired wall remote...
